server/tests/src/test/java/org/infinispan/server/functional/memcached/MemcachedOperations.java

```diff
@@ -105,13 +105,31 @@ public void testConcurrentGets(ConnectionFactoryBuilder.Protocol protocol) throw
       }
    }
 
+   @ParameterizedTest
+   @EnumSource(ConnectionFactoryBuilder.Protocol.class)
+   public void testImplicitNoAuthMemcached(ConnectionFactoryBuilder.Protocol protocol)  {
+      ConnectionFactoryBuilder builder = new ConnectionFactoryBuilder();
+      builder.setProtocol(protocol);
+      MemcachedClient client = SERVERS.memcached().withClientConfiguration(builder).withPort(11222).get();
+      try {
+         client.get(k());
+         if (protocol == ConnectionFactoryBuilder.Protocol.TEXT) {
+            throw new RuntimeException("Implicit memcached text request should have failed");
```

Comment:
This exception can never be propagated due to the catch block. Should it not propagate? If not then I would say there is no reason to throw the exception.

Reply:
I've made the caught exception more specific